### PR TITLE
Fix: Do not use properties file

### PR DESCRIPTION
**Pull request by mariobodemann**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -193,7 +194,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## Version 1.0.0 - 2014-08-13
 Initial release.
 
-[unreleased]: https://github.com/contentful/contentful.java/compare/java-sdk-8.0.0...HEAD
+[unreleased]: https://github.com/contentful/contentful.java/compare/java-sdk-8.1.0...HEAD
+[8.1.0]: https://github.com/contentful/contentful.java/compare/java-sdk-8.0.0...java-sdk-8.1.0
```
Comment (loudmouth) [inline CHANGELOG.md]: I'm a bit confused, why is 8.1.0 marked as unreleased? isn't the plan to release it on merge of this PR?

File: src/test/java/com/contentful/java/cda/ClientTest.java [hunk 2]
```diff
@@ -148,8 +149,10 @@ public void authHeader() throws Exception {
   @Test
   @Enqueue
   public void userAgentHeader() throws Exception {
-    String versionName = Util.getProperty("version.name");
+    String versionName = GeneratedBuildParameters.PROJECT_VERSION;
     assertThat(versionName).isNotEmpty();
+    assertThat(versionName).doesNotContain("${");
+
```
Comment (loudmouth) [inline src/test/java/com/contentful/java/cda/ClientTest.java]: could we use a regular expression to test the version number?

File: CHANGELOG.md (hunk 1 above)
Comment (mariobodemann) [reply CHANGELOG.md]: The 8.1.0 is not yet released. Before I do the releasing I'll test it manually and add a date and remove the flag.

File: src/test/java/com/contentful/java/cda/ClientTest.java (hunk 2 above)
Comment (mariobodemann) [reply src/test/java/com/contentful/java/cda/ClientTest.java]: Yes please!

File: src/test/java/com/contentful/java/cda/ClientTest.java [hunk 3]
```diff
@@ -148,8 +149,9 @@ public void authHeader() throws Exception {
   @Test
   @Enqueue
   public void userAgentHeader() throws Exception {
-    String versionName = Util.getProperty("version.name");
-    assertThat(versionName).isNotEmpty();
+    String versionName = GeneratedBuildParameters.PROJECT_VERSION;
+    assertThat(versionName).matches("^\\d+\\.\\d+\\.\\d+(-\\w+)?$");
```
Comment (loudmouth) [inline src/test/java/com/contentful/java/cda/ClientTest.java]: 👍

**Pull request merged**